pom.xml

```diff
@@ -216,7 +216,7 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
-            <version>2.6.1</version>
+            <version>2.7.1</version>
```

Comment:
Bumped the hsqldb plugin while working on debugging invalid cast exceptions.  Not required, but gonna leave it.